### PR TITLE
Fix the Jenkins attribute mapping and other important standard variables for CLI and exclude few more sensitive information

### PR DESCRIPTION
**Pull request by hemanthgk10**
Signed-off-by: Hemanth Gokavarapu <hemanth@soluble.ai>

## JIRA ISSUE
https://lacework.atlassian.net/browse/RAIN-37274

## DESCRIPTION
Fixing the attribute mapping based on the information from Teladoc Jenkins CLI run.

## TESTING
- Added the required test cases to make sure that right variables are not being filtered and filter out the sensitive information

### CODE REVIEW

File: pkg/xcp/xcp_test.go [hunk 1]
```diff
@@ -44,6 +44,46 @@ func TestGetCIEnv(t *testing.T) {
 	os.Setenv("ATLANTIS_TERRAFORM_VERSION", "yyy")
 	os.Setenv("PULL_NUM", "yyy")
 	os.Setenv("REPO_REL_DIR", "yyy")
+	os.Setenv("BUILD_ID", "27")
+	os.Setenv("JOB_BASE_NAME", "main")
+	os.Setenv("KUBERNETES_PORT", "tcp://172.20.0.1:443")
+	os.Setenv("RUN_ARTIFACTS_DISPLAY_URL", "https://ci.intouchhealth.io/")
+	os.Setenv("TF_VAR_adminpassword", "****")
+	os.Setenv("TF_VAR_adminusername", "***")
+	os.Setenv("ARM_TENANT_ID", "****")
```
Comment (satellite15) [inline pkg/xcp/xcp_test.go:53]: nit: We don't have a test for the CLIENT substring

**Review by satellite15 — APPROVED** [review]
Just on nit on the test

File: pkg/xcp/xcp_test.go (hunk 1 above)
Comment (hemanthgk10) [reply pkg/xcp/xcp_test.go:54]: sorry pushed the change. fixed it.

**Comment by hemanthgk10** [comment]
@satellite15 Added that missing test case as well.. now

**Pull request merged**